lucetc/src/compiler.rs

```diff
@@ -196,14 +197,19 @@ impl<'a> Compiler<'a> {
             })
             .collect();
 
+        let start = Instant::now();
```

Comment:
and right after putting up the PR I realize... I thought at first that Cranelift recorded timings with a global and some macro'd over not-actually-unsafe unsafe, but it's all thread local refcells. I should do the same here.